.github/workflows/test-db-ops.yaml

```
@@ -25,15 +27,125 @@ on:
         required: true
         default: 'dev'
-      tag:
-        type: string
-        description: 'Image Tag'
-        required: true
-        default: ''
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository_owner }}/workflows-service
+  SHORT_HASH: ${{ github.event.client_payload.environment == 'prod' && github.env.PROD_WF_SHORT_SHA || github.event.client_payload.environment == 'sb' && github.env.SB_WF_SHORT_SHA || github.env.DEV_WF_SHORT_SHA }}
```

actionlint

receiver of object dereference "prod_wf_short_sha" must be type of object but got "string"

(expression)